src/component/events.ts

```diff
@@ -85,23 +87,28 @@ export const scheduleProcessing = internalMutation({
 
 export const processEvents = internalAction({
   args: {
-    sdkKey: v.string(),
     options: eventsOptions,
   },
-  handler: async (ctx, { sdkKey, options }) => {
+  handler: async (ctx, { options }) => {
     const events = await ctx.runQuery(internal.events.getOldestEvents, {
       count: EVENT_BATCH_SIZE,
     });
 
+    const sdkKey = await ctx.runQuery(internal.sdkKey.get);
```

Comment:
ditto - can avoid subquery

Reply:
This one's in an action so the subquery is necessary